google-built-opentelemetry-collector/Dockerfile

```diff
@@ -25,5 +25,5 @@ USER ${USER_UID}
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --chmod=755 otelcol-google /otelcol-google
 COPY config.yaml /etc/otelcol-google/config.yaml
-ENTRYPOINT ["/otelcol-google", "--feature-gates=exporter.googlemanagedprometheus.intToDouble"]
+ENTRYPOINT ["/otelcol-google", "--feature-gates=exporter.googlemanagedprometheus.intToDouble,exporter.googlecloud.CustomMonitoredResources"]
```

Comment:
nit: include a space inbetween each comma

Reply:
This is generated using distrogen, also the docs say no spaces between them: https://github.com/open-telemetry/opentelemetry-collector/tree/main/featuregate#controlling-gates